roles/ssp/templates/config/authsources-1.14.php.j2

```diff
@@ -4,6 +4,10 @@
  * {{ ansible_managed }}
  */
 
+{% if ssp_authsources_function_raw is defined %}
+{{ ssp_authsources_function_raw }}
```

Comment:
rename to ssp_authsources_preamble
and add it to defaults/main.yml (in comments)

```
# Added before the config[] in SimpleSAMLphp authentication sources
#ssp_authsources_preamble: |
#    function foo() {
```

Reply:
Done